localDev/index.js

```diff
@@ -0,0 +1,35 @@
+
+function createScene(engine) {

```

Comment:
index.js should not be merged into the repo

Reply:
reverted